lib/packages.cf

```diff
@@ -177,6 +180,14 @@ body package_module freebsd_ports
     query_updates_ifelapsed => "1440";
 }
 
+body package_module msiexec
```

Comment:
This needs to be somehow guarded with `if` macro to be skipped on versions that don't support `interpreter` and `module_path`.

Reply:
(look at the Travis results)

Reply:
I believe that the if macro does not support patch release, only major minor. I can look into dynamic inclusion, maybe move it to a file that is only included if running specific versions

Reply:
@nickanderson when you have time, can you please follow up on this?

Reply:
@Lex-2008 I opened a PR to your PR. 

https://github.com/Lex-2008/masterfiles/pull/4
I found that ENT-3366 is fixed in 3.10.0, it doesn't work in 3.7.8 though, so this would introduce breakage for backward compatibility in masterfiles 3.12.x to 3.7.x.

I think it would be better than going down the path of version based policy loading again. It was a long process to get from version based stdlib back to it all being compatible with any supported version.

Reply:
Here is another option that should be compatible back to 3.7 https://github.com/Lex-2008/masterfiles/pull/5 (so tests should pass)